src/main/resources/application.yml

```diff
@@ -21,6 +21,7 @@ spring:
             token-uri: \${AUTH_CLIENT_TOKEN_URI:}
             user-name-attribute: sub
             jwk-set-uri: \${AUTH_CLIENT_JWK_URI:}
+white-list: \${WHITE_LIST_USERS:}
```

Comment:
can you give to this property a more significant name? this is a white list of what?

Reply:
renamed